contracts/disputes/DisputeManager.sol

```diff
@@ -154,6 +154,15 @@ contract DisputeManager is DisputeManagerV1Storage, GraphUpgradeable, IDisputeMa
         _;
     }
 
+    modifier onlyPendingDispute(bytes32 _disputeID) {
+        require(isDisputeCreated(_disputeID), "Dispute does not exist");
```

Comment:
Seems like this line can be removed, as the check for Pending will also mean checking for not Null.